### DISCUSSION TITLE
Application data storage issue

### DISCUSSION THREAD

**Discussion by zero5652pzj** [deta/space-docs#43] [General]
I installed App.I would like to know where the data will be saved? Can it be saved in my collections?

**Comment by sudo-subg** [comment]
Every Space app gets dedicated database and file storage services. All data stored by a Space app is contained within the app instance. Simply click the context menu `...` button on the app tile and choose `View Data` to access the data of your app's Base and Drive instances.


> Can it be saved in my collections?

As of now, no

**Comment by zero5652pzj** [comment]
I understand now. Data isolation alone ensures security which is great. I really like your product and hope it will keep getting better.



在 2023-03-08 16:56:34，"Bighneswar Parida" ***@***.***> 写道：

Every Space app gets dedicated database and file storage services. All data stored by a Space app is contained within the app instance. Simply click the context menu ... button on the app tile and choose View Data to access the data of your app's Base and Drive instances.

Can it be saved in my collections?

As of now, no
